themes/ddbasic/sass/components/node/paragraphs.scss

```diff
@@ -0,0 +1,171 @@
+//
+// Styling of paragraphs
+
+@import '../../base.scss';
+
+// Paragraphs block display
+.paragraphs-block {
+  position: relative;
+  display: inline-block;
+  margin: 0 0 $distance-paragraph;
+  width: 100%;
+
+  &.half_left {
```

Comment:
Wouldn't it be better replicate the approach used in the rest of the theme?
I don't mind rewriting it but no other css file in the theme uses the BEM structure so that would make this file stand out.

Reply:
The problems with the current approach has been discussed at length and [we have tried to set a better standard for major changes going forward](https://github.com/ding2/ding2/pulls?utf8=%E2%9C%93&q=is%3Apr+BEM+). That being said I understand any reluctance to changing this.

Reply:
I have added the suggested classes paragraphs-block--half-left, paragraphs-block--half-right, and a few more where it makes sense. This has reduced the nesting considerably safe from the .ting-object part thats somewhat more complex to alter.

https://github.com/ding2/ding2/pull/1406/commits/3ccdc857b15d2d72e933f59d71b5cde664db9dea